webpack.config.js

```diff
@@ -1,7 +1,10 @@
 const path = require('path');
 const webpack = require('webpack');
+const { VueLoaderPlugin } = require("vue-loader");
+
 module.exports = {
   entry: './src/main.js',
+  mode: "development",
```

Comment:
We could use `process.env.NODE_ENV || 'development'` for the `mode` and then we don't need to additionally supply it as a parameter in the build script.

Reply:
Much better.  Changed here: 2a88f3f139397f3180f28466a3368f4a83c29751